### ISSUE TITLE
Volume stuck in deleting

### ISSUE THREAD

**Issue by cr0** [longhorn/longhorn#1011]
Hi, one of my volumes is stuck in the deleting state for days. The volume is released but it is now in a failed state for the last remaining node. Also it is assigned to an instance manager which does not exist anymore. 
![deleting](https://user-images.githubusercontent.com/854965/72166395-6ee27780-33c9-11ea-8bcc-540428c047bc.png)

Smashing the delete button does not help ;) I also rebooted the assigned node, no changes. Dropdown options are: Backups, Delete, Create PV/PVC

**Comment by yasker** [comment]
@cr0 Can you send us a support bundle at `longhorn-support-bundle@rancher.com` ? We will look into it. It looks like something is preventing the volume from deleting (probably the instance manager).

**Comment by cr0** [comment]
@yasker Is there any senstive data in the support bundle. Have to make sure I am allowed to forward it.

**Comment by yasker** [comment]
@cr0 I don't believe so. It's a zip file contains the log of the pods in the `longhorn-system` namespace, as well as some metadata used by Longhorn. You can unzip the file and take a look to make sure it doesn't contain any sensitive data.

**Comment by derekbit** [comment]
Feel free to reopen the ticket if it is valid in newer versions (v1.6+).
